rust/connlib/shared/src/callbacks.rs

```diff
@@ -88,6 +88,7 @@ pub struct ResourceDescriptionDns {
     pub sites: Vec<Site>,
 
     pub status: Status,
+    pub disableable: bool,
```

Comment:
nit: `can_be_disabled` may be easier to read than `disableable` or `disablable`

Reply:
Other alternatives:

- `can_toggle`
- `can_bypass`
- `deactivatable`
- `disconnectable` (as in, we allow the user to "disconnect" a resource)